java/maven/src/org/netbeans/modules/maven/classpath/ClassPathProviderImpl.java

```diff
@@ -538,10 +539,17 @@ public ClassPath getActiveClassPath() {
             if (ret == null) {
                 // see org.apache.maven.plugin.compiler.CompilerMojo.classpathElements
                 for (String sourceRoot : proj.getOriginalMavenProject().getCompileSourceRoots()) {
-                    if(new File(sourceRoot, MODULE_INFO_JAVA).exists()) {
-                        ret = hasModuleInfoCP.get();  
+                    final File moduleInfoFile = new File(sourceRoot, MODULE_INFO_JAVA);
```

Comment:
Seems the new code is not setting "ret" at all, and this whole section now, at least at the first glance, seems to have no effect?

Reply:
Thanks @jlahoda , so this code "worked" when either (or both) the IDE and project used jdk1.8 since you don't want module-info.java to be used (AFAICT). There's a problem when both IDE and project are jdk-11, checking...